packages/multichain-network-controller/package.json

```diff
@@ -68,7 +68,8 @@
     "typescript": "~5.2.2"
   },
   "peerDependencies": {
-    "@metamask/network-controller": "^22.2.1"
+    "@metamask/accounts-controller": "^23.0.0",
```

Comment:
This one was missing, and I added it to "enforce" the relation that the `multichain-network-controller` and the `accounts-controller` share.

Unfortunately, we cannot express this on the `accounts-controller` (or at least, IDK how to express that for now)

Reply:
Can you elaborate on why this is missing? I don't see `accounts-controller` being referred to anywhere in `multichain-network-controller`.

Reply:
Ah, I missed that we are just referring directly to the event name here, sorry: https://github.com/MetaMask/core/blob/fe244b1f1fa300cb9ad98e2ac229a31d1cb7f4a2/packages/multichain-network-controller/src/MultichainNetworkController.ts#L192.

Maybe we ought to import the event name from the package at the top so it's easy to spot?

Reply:
We definitely should, yes!